tests/ci/report.py

```diff
@@ -167,6 +172,78 @@
 BASE_HEADERS = ["Test name", "Test status"]
 
 
+@dataclass
+class TestResult:
+    name: str
+    status: str
+    # the following fields are optional
+    time: Optional[float] = None
+    log_files: Optional[List[Path]] = None
+    raw_logs: Optional[str] = None
```

Comment:
Raw logs of what? Just not clear why it is not a List. 

I would add better comments to show what is going where in report. It can be a link to docs with image and corresponding names for simplicity.

Reply:
It's a text, raw logs per test. LIke in stateless tests